packages/account/src/Sections/Security/Passkeys/passkeys.tsx

```diff
@@ -1,20 +1,57 @@
 import React from 'react';
 import { Redirect } from 'react-router-dom';
 import { Loading } from '@deriv/components';
-import { useGetPasskeysList, useIsPasskeySupported } from '@deriv/hooks';
+import { useGetPasskeysList, useIsPasskeySupported, useRegisterPasskey } from '@deriv/hooks';
 import { PlatformContext, routes } from '@deriv/shared';
 import { observer, useStore } from '@deriv/stores';
 import NoPasskeysSet from './no-passkeys-set';
 import './passkeys.scss';
+import PasskeysStatus from 'Sections/Security/Passkeys/passkeys-status';
+import { Localize } from '@deriv/translations';
+import PasskeysList from 'Sections/Security/Passkeys/passkeys-list';
+
+const mock_passkeys_list = [
+    {
+        id: 1,
+        name: 'Passkey Name 1',
+        last_used_at: 1702365923,
+        created_at: 1702365923,
+        stored_on: 'no info',
+    },
+    {
+        id: 2,
+        name: 'Passkey Name 2',
+        last_used_at: 1702365923,
+        created_at: 1702365923,
+        stored_on: 'no info',
+    },
+    {
+        id: 3,
+        name: 'Passkey Name 2',
+        last_used_at: 1702365923,
+        created_at: 1702365923,
+        stored_on: 'no info',
+    },
+];
 
 const Passkeys = observer(() => {
     const { ui } = useStore();
     const { is_mobile } = ui;
     const { is_passkeys_enabled } = React.useContext(PlatformContext);
+
+    const [should_show_status, setShouldShowStatus] = React.useState(false);
+
     const { is_passkey_supported, is_loading: is_passkey_support_checked } = useIsPasskeySupported();
     const should_show_passkeys = is_passkeys_enabled && is_passkey_supported && is_mobile;
 
     const { data: passkeys_list, isLoading: is_passkeys_list_loading } = useGetPasskeysList();
+    const { createPasskey, is_registration_in_progress, is_passkey_registered } = useRegisterPasskey();
+
+    React.useEffect(() => {
+        if (is_passkey_registered) {
+            setShouldShowStatus(true);
+        }
+    }, [is_passkey_registered]);
 
     if (is_passkey_support_checked || is_passkeys_list_loading) {
```

Comment:
nitpick, can we rename `is_passkey_support_checked` to something like `is_passkey_support_loading` makes it more readable 😄

Reply:
I'll go with 'is_passkey_support_checking' if you don't mind :)